unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatController.cs

```diff
@@ -206,19 +215,48 @@ public void UpdateTotalUnseenMessagesByUser(string json)
         }
     }
 
+    // called by kernel
+    [UsedImplicitly]
+    public void UpdateChannelUnseenMessages(string json)
```

Comment:
what do you think if we remove this message and just leave the option of sending a list here: `UpdateTotalUnseenMessagesByChannel`

Reply:
Yes, we could do it perfectly. But being a change of our data design, I prefer to wait for next monday to decide together.